cli/src/cortx_prep.sh

```diff
@@ -44,8 +44,11 @@ function install_prvsnr() {
         mount -t iso9660 /opt/isos/${cortx_iso} /tmp/iso_mount/cortx 2>&1 | tee -a ${LOG_FILE}
         mount -t iso9660 /opt/isos/${os_iso} /tmp/iso_mount/cortx-os 2>&1 | tee -a ${LOG_FILE}
 
+        echo "INFO: Backing up exisitng repositories" 2>&1 | tee -a ${LOG_FILE}
```

Comment:
Update this line to:
`echo "INFO: Backing up exisitng repositories" 2>&1 | tee -a "${LOG_FILE}"`

To fix Codacy error.

Reply:
Fixed